contributing/DEVELOPING.md

```diff
@@ -65,16 +65,17 @@ The Quick Start Guide's  [One-time setup](getting-started-quick.md#one-time-setu
 
 For manual testing build AMP and start the Node.js server by running `gulp`.
 
-### Examples
-
-The content in the `examples` directory can be reached at: http://localhost:8000/examples/
+### Serve Mode
+There are 3 serving modes:
+- MAX mode serves unminified AMP. You want to use this during normal dev. `gulp` serves MAX mode by default.
+- MIN mode serves minified AMP. This is closer to the prod setup. This is only avaialable after running `gulp dist --fortesting`. Serve MIN mode by adding `--compiled` to `gulp` command.
+- CDN mode serves prod. This file would not reflect your local changes. Serve CDN mode by adding `--cdn` to `gulp` command.
 
-For each example there are 3 modes:
+To switch serving mode in runtime. please to to http://localhost:8000/serve_mode=$mode. $mode can be `max/min/cdn`.
```

Comment:
Rewrite as:

To switch serving modes during runtime, go to http://localhost:8000/serve_mode=$mode and set the `$mode` to one of the following values: `max`, `min`, or  `cdn`.